packages/frontend/hooks/countDownTimer.ts

```diff
@@ -0,0 +1,32 @@
+﻿import { useEffect, useState } from 'react';
+
+const useCountdown = (targetDate: Date) => {
+  const countDownDate = new Date(targetDate).getTime();
+
+  const [countDown, setCountDown] = useState(
+    countDownDate - new Date().getTime(),
+  );
+
+  useEffect(() => {
+    const interval = setInterval(() => {
+      setCountDown(countDownDate - new Date().getTime());
+    }, 1000);
+
+    return () => clearInterval(interval);
+  }, [countDownDate]);
```

Comment:
is it triggered on every page render? And what if 0 0 0 0? Maybe we can reuse our timer hook